docs/guides/burst_insar_product_guide.md

```diff
@@ -250,6 +252,47 @@ The text file with extension .txt includes processing parameters used to generat
 
 *Table 4: List of InSAR parameters included in the parameter text file*
 
+### Merge Sentinel-1 Burst InSAR Products
+
+Burst InSAR products created using the insar_tops_burst workflow can be merged together using the merge_tops_burst workflow. This can be useful when the deformation signal you'd like to observe spans multiple bursts. Suppose users have already installed the [hyp3-isce2 plugin]( https://github.com/ASFHyP3/hyp3-isce2 "HyP3-ISCE2 Plugin" ){target=_blank} on their local machine, merge can be done using the following syntax:
+
+python -m hyp3_isce2 ++process merge_tops_bursts PATH_TO_UNZIPPED_PRODUCTS --apply-water-mask True
+
+Where PATH_TO_UNZIPPED_PRODUCTS is the path to a directory containing unzipped burst InSAR products. For example:
+
+PATH_TO_UNZIPPED_PRODUCTS
+
+├─ S1_136232_IW2_20200604_20200616_VV_INT80_663F
+
+├─ S1_136231_IW2_20200604_20200616_VV_INT80_529D
+
+In order to be merging eligible, all burst products must:
+
+Have the same reference and secondary dates
+Have the same polarization
+Have the same multilooking
+Be from the same relative orbit
+Be contiguous
+
+The workflow should through an error if any of these conditions are not met.
+
+Merging burst InSAR products requires extra data that is not contained in the production HyP3 Burst InSAR products. For the time being, to be merging eligible burst products must be created locally using your own installation of hyp3-isce2 from the merge_bursts branch of this repository!
+
+As mentioned above this feature is under active development, so we welcome any feedback you have!
+
+Options
+To learn about the arguments for each workflow, look at the help documentation (python -m hyp3_isce2 ++process [WORKFLOW_NAME] --help).
+
+Looks Option
+When ordering Sentinel-1 Burst InSAR On Demand products, users can choose the number of looks (--looks) to use in processing, which drives the resolution and pixel spacing of the output products. The available options are 20x4, 10x2, or 5x1. The first number indicates the number of looks in range, the second is the number of looks in azimuth.
+
+The output product pixel spacing depends on the number of looks in azimuth: pixel spacing = 20 * azimuth looks
+
+Products with 20x4 looks have a pixel spacing of 80 m, those with 10x2 looks have a pixel spacing of 40 m, and those with 5x1 looks have a pixel spacing of 20 m.
```

Comment:
```suggestion
Products with `20x4` looks have a pixel spacing of 80 m, those with `10x2` looks have a pixel spacing of 40 m, and those with `5x1` looks have a pixel spacing of 20 m.
```

Reply:
Could this also be expressed as a table? for example: 

| Looks | Pixel Spacing |
| --- | --- |
| `20x4` | 80 m |
| `10x2` | 40 m | 
| `5x1` | 20 m |